src/main/java/org/broadinstitute/hellbender/tools/reference/ReferenceSequenceTable.java

```diff
@@ -0,0 +1,354 @@
+                    md5 = ReferenceUtils.calculateMD5(source, referenceInterval);
+                }
+                else {
+                    md5 = md5FromDict;
+                }
+                break;
+            case ALWAYS_RECALCULATE:
+                md5 = ReferenceUtils.calculateMD5(source, referenceInterval);
+                break;
+        }
+        return md5;
+    }
+
+    /**
+     * Generate ReferencePairs for pairwise comparison of all references present in the table
+     *
+     * @return the list of ReferencePairs for every pair of references
+     */
+    public List<ReferencePair> generateReferencePairs(){
+        List<ReferencePair> referencePairs = new ArrayList<>();
+        for(int i = 0; i < references.size(); i++){
+            for(int j = i + 1; j < references.size(); j++){
+                referencePairs.add(new ReferencePair(this, references.get(i), references.get(j)));
+            }
+        }
+        return referencePairs;
+    }
+
+    /**
+     * Analyze the table by doing a pairwise comparison for all table references. Generates all ReferencePairs, then analyzes
+     * each pair and assigns it an analysis as a set of the following statuses:
+     *      EXACT_MATCH,
+     *      DIFFER_IN_SEQUENCE_NAMES,
+     *      DIFFER_IN_SEQUENCE,
+     *      DIFFER_IN_SEQUENCES_PRESENT,
+     *      SUPERSET,
+     *      SUBSET
+     *
+     * @return list of ReferencePairs with updated status sets
+     */
+    public List<ReferencePair> analyzeTable(){
+        List<ReferencePair> refPairs = generateReferencePairs();
+
+        for(TableRow row : tableByMD5.values()) {
+            for(ReferencePair pair : refPairs) {
+                int ref1Index = pair.getRef1ColumnIndex();
+                int ref2Index = pair.getRef2ColumnIndex();
+                TableEntry[] entries = row.getEntries();
+
+                TableEntry ref1Value = entries[ref1Index];
+                TableEntry ref2Value = entries[ref2Index];
+                if (!ref1Value.getColumnValue().equals(ref2Value.getColumnValue())) {
+                    pair.removeStatus(ReferencePair.Status.EXACT_MATCH);
+                }
+                if(!ref1Value.getColumnValue().equals(ref2Value.getColumnValue()) &&
+                        !(ref1Value.getColumnValue().equals(MISSING_ENTRY) || ref2Value.getColumnValue().equals(MISSING_ENTRY))){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCE_NAMES);
+                }
+            }
+        }
+        for(ReferencePair pair : refPairs){
+            boolean superset = false;
+            boolean subset = false;
+            for(String sequenceName : tableBySequenceName.keySet()){
+                int ref1Index = pair.getRef1ColumnIndex();
+                int ref2Index = pair.getRef2ColumnIndex();
+                Set<ReferenceSequenceTable.TableRow> rows = queryBySequenceName(sequenceName);
+                int sequenceNameFound = 0;
```

Comment:
This is really counting the number of times a sequence name was found *and* the other reference was missing the sequence. Consider renaming to make this clearer. Eg., `sequenceNameFoundInOneRef`